Mathlib/MeasureTheory/Measure/Haar/Basic.lean

```diff
@@ -698,6 +698,14 @@ theorem haarMeasure_unique (μ : Measure G) [SigmaFinite μ] [IsMulLeftInvariant
 #align measure_theory.measure.haar_measure_unique MeasureTheory.Measure.haarMeasure_unique
 #align measure_theory.measure.add_haar_measure_unique MeasureTheory.Measure.addHaarMeasure_unique
 
+/-- Let `K₀ K₁ : TopologicalSpace.PositiveCompacts G`, then `K₁` defines the same Haar measure as
+`K₀` iff it has measure `1` for the Haar measure defined by `K₀`. -/
+@[to_additive]
+theorem haarMeasure_eq_iff (K₀ K₁ : PositiveCompacts G) :
+    haarMeasure K₀ = haarMeasure K₁ ↔ (haarMeasure K₀) K₁ = 1 :=
```

Comment:
What do you think of replacing `haarMeasure K₀` by an arbitrary Haar measure here? It would still apply to this case, but I think it would also be useful in other cases. For example I think it would simplify the proof of `Basis.map_addHaar`.

Reply:
That's a really good point! I didn't think about it but it should make things a lot cleaner.

Reply:
The results are now stated with the generality of [MeasureTheory.Measure.haarMeasure_unique](https://leanprover-community.github.io/mathlib4_docs/Mathlib/MeasureTheory/Measure/Haar/Basic.html#MeasureTheory.Measure.haarMeasure_unique)